rotkehlchen/externalapis/opensea.py

```diff
@@ -246,12 +244,12 @@ def _deserialize_nft(
             collection = None
             # NFT might not be part of a collection
             if 'collection' in entry:
-                saved_entry = self.collections.get(entry['collection']['name'])
+                saved_entry = self.collections.get(entry['collection'])
```

Comment:
At line 259, `entry['collection']` is reused.

And we have some keys that won't work the way they are there:

```
                        banner_image=collection_data['banner_image_url'],
                        description=collection_data['description'],	                        description=collection_data['description'],
                        large_image=collection_data['large_image_url'],	                        
                        large_image=collection_data['large_image_url'],
```

Reply:
raising UnknownAsset error instead